plugins/webhooks/index.js

```diff
@@ -310,6 +311,94 @@ function pullRequestEvent(pluginOptions, request, reply, parsed) {
         .catch(err => reply(boom.wrap(err)));
 }
 
+/**
+ * Check the pipeline have triggered job or not
+ * @method  isTrigger
+ * @param   {Pipeline}  pipeline    The pipeline to check
+ * @param   {String}    startFrom   The trigger name
+ * @returns {boolean}               If true the pipeline has triggered job
+ */
+function isTrigger(pipeline, startFrom) {
+    const nextJobs = workflowParser.getNextJobs(pipeline.workflowGraph, {
+        trigger: startFrom
+    });
+
+    return nextJobs.length > 0;
+}
+
+/**
+ * Get all pipelines which has triggered job
+ * @method  triggerPipelines
+ * @param   {pipelineFactory}   pipelineFactory the pipeline factory to get branch list
+ * @param   {Object}            scmConfig       has the token and scmUri to get branches
+ * @param   {String}            branch          the branch which is committed
+ * @returns {Promise}                           Promise that resolves into triggered pipelines
+ */
+function triggerPipelines(pipelineFactory, scmConfig, branch) {
+    return pipelineFactory.scm.getBranchList(scmConfig)
+        .then((branches) => {
+            const scmUris = [];
+            const splitUrl = scmConfig.scmUri.split(':');
+
+            branches.forEach((b) => {
```

Comment:
How about using `map()` like https://github.com/screwdriver-cd/scm-github/pull/91#discussion_r192939228

Reply:
It is better to use map(). I will fix.